cawbird: 1.3.2 -> 1.4.1

schmittlauch opened the pull request:
###### Motivation for this change

- regular minor feature update, changelog: https://github.com/IBBoard/cawbird/releases/tag/v1.4
- specified exact version of GPLv3 usage

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!
List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://nixos.org/manual/nixpkgs/unstable/#chap-reviewing-contributions
-->


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

schmittlauch committed:
cawbird: 1.3.2 -> 1.4.0

- regular minor feature update, changelog: https://github.com/IBBoard/cawbird/releases/tag/v1.4
- specified exact version of GPLv3 usage

schmittlauch commented:
@GrahamcOfBorg build cawbird

SuperSandro2000 commented on pkgs/applications/networking/cawbird/default.nix:
```diff
@@ -69,11 +69,13 @@ stdenv.mkDerivation rec {
     patchShebangs data/meson_post_install.py
   '';
 
+  mesonFlags = [ "-Dconsumer_key_base64=VmY5dG9yRFcyWk93MzJEZmhVdEk5Y3NMOA==" "-Dconsumer_secret_base64=MThCRXIxbWRESDQ2Y0podzVtVU13SGUyVGlCRXhPb3BFRHhGYlB6ZkpybG5GdXZaSjI=" ];
```
```suggestion
  mesonFlags = [
    "-Dconsumer_key_base64=VmY5dG9yRFcyWk93MzJEZmhVdEk5Y3NMOA=="
    "-Dconsumer_secret_base64=MThCRXIxbWRESDQ2Y0podzVtVU13SGUyVGlCRXhPb3BFRHhGYlB6ZkpybG5GdXZaSjI="
  ];
```

schmittlauch replied:
I guess I'll just throw nixpkgs-fmt on it.

SuperSandro2000 reviewed (commented):
Where di you get the keys from?

schmittlauch commented:
@SuperSandro2000 These are the default keys from upstream as provided in https://github.com/IBBoard/cawbird/blob/master/README.md#preparation.
It'd be nice to have separate keys for the nixpkgs build, but only Twitter accounts with a linked phone number are able to request API keys which I'm unwilling to do. So unless anyone else provides new keys, we're gonna use the ones from upstream despite all the app-level rate limiting.

SuperSandro2000 commented:
Can you add a comment with a short explanation and the link? That would be great.

schmittlauch commented:
:ballot_box_with_check:

schmittlauch committed:
cawbird: 1.4 -> 1.4.1

- hotfix release
- changelog: Hotfix to correct "Send" button state issues for direct messages

schmittlauch commented:
In the meantime, a small hotfix got released. I put that update step into this PR as well.
